.openpublishing.redirection.json

```diff
@@ -2621,6 +2620,31 @@
             "source_path": "docs/visual-basic/reference/vb6-support.md",
             "redirect_url": "/visualstudio/vb6/vb6-support",
             "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/core/docker/index.md",
+            "redirect_url": "intro-net-docker",
```

Comment:
```suggestion
            "redirect_url": "/dotnet/core/docker/intro-net-docker",
```

Reply:
Makes sense, do you think this is a bug with the cleanup script, or was the old redirect in an incorrect format?

Reply:
I don't know, but I have a feeling that *index.md* files shouldn't be redirected, unless the whole TOC branch is gone. Existing *docs/core/docker/index.md* looks like preparation for future fill-up.

Reply:
Looks like it used to have something, but got clear out in 0d91df73136027c2f554222d827d653163abbacd

I've rebased out the non-redirect changes, and included your fix

Reply:
@Thraka can you comment?

Reply:
I renamed the index.md file to a descriptive file name since file names impact seo

Reply:
@Thraka you did the redirect correctly. The following link works as expected:
https://docs.microsoft.com/en-us/dotnet/core/docker/index

That is the clean-up tool that doesn't handle it correctly: Microsoft/vscode-docs-authoring#202

I think the question is *where* that redirect should reside:
  - in the global (master) redirection *json* file (as suggested by this PR)
  - or in the local *index.md* file (as it is now)

Reply:
The answer is: in the master redirection file. 

@Thraka We don't usually rename our index.md files to some other thing just for SEO. They're the main gateway to each folder and the path is usually good enough already. In this case, the change is actually breaking some of our SEO rules: it duplicates words in the path and says intro instead of introduction.

Note that index is not indexed as part of the path:
![image](https://user-images.githubusercontent.com/12971179/62187773-4ab14380-b31f-11e9-976e-9e71f8e4a549.png)

Reply:
Also, I'd probably roll back the redirect change in this case.

Reply:
Hrm yes, I'll rename that file and do a new redirect.